pomerolPackages/pomerol/default.nix

```diff
@@ -13,27 +13,19 @@
 
 stdenv.mkDerivation rec {
   pname = "pomerol";
-  version = "2.0";
+  version = "2.2";
 
   src = fetchFromGitHub {
-    owner = "aeantipov";
+    owner = "pomerol-ed";
     repo = pname;
     rev = version;
-    sha256 = "sha256-RlpcPVsLuvMeECpVp92nODDij9+tciySdUCOqrqFtvA=";
+    sha256 = "sha256-s65GSNZcV2virhc6WquhhdNs7YojendQSDHQycSsi7I=";
   };
 
-  patches = [
-    (fetchpatch {
-      name = "fix-quantum_model-is-not-a-valid-prog.patch";
-      url = "https://github.com/aeantipov/pomerol/commit/a7dee04938584c05dd827becd906e4a1483f5b53.patch";
-      sha256 = "sha256-/QEzacrSnSsxeyjAHVH0N3bJFxQf2TgfGl88MqBctQ0=";
-    })
-    (fetchpatch {
-      name = "CI-Update-value-of-libcommute_DIR-CMake-var-after-kr.patch";
-      url = "https://github.com/aeantipov/pomerol/commit/6de692ba27109972b279085f6c2bf6bbad2fc986.patch";
-      sha256 = "sha256-iI+E8oFuasFf5KIc/TvyisOarSY5nEb1/o9QfLGGexM=";
-    })
-  ];
+  # The tests will fail, if it is impossible to spawn 16 mpi processes
+  # this patch will limit it to a reasonable 4
+  patches = [ ./mpitest.patch ];
```

Comment:
Why is `OMPI_MCA_rmaps_base_oversubscribe = "yes";` (further down in the file) not enough?